src/test.js

```diff
@@ -491,6 +493,13 @@ function synchronize( callback, priority ) {
 
 	if ( priority ) {
 		config.queue.splice( priorityCount++, 0, callback );
+	} else if ( random ) {
+		if ( !randomNumber ) {
+			randomNumber = randomNumberGenerator( random );
```

Comment:
Hmm, I'll explore more options. Not sure how I feel about `unitSample` as it seems a bit vague (but maybe that's just me).